src/pages/WishList/Empty.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Empty = () => {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/Main');
+  };
+
+  return (
+    <div className="cartBaseContainer">
+      <div className="cartContainer">
+        <div className="cartContainer empty">
+          <div className="emptyCartImage">
+            <img alt="emptycart" src="images/icon/shopping-cart.png" />
```

Comment:
```suggestion
            <img alt="emptycart" src="/images/icon/shopping-cart.png" />
```
절대경로로 적어주세요! 상대경로로 지정하게 되면 path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!

예시) 상대경로로 작성했을 시
- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.